qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -380,6 +368,26 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
     <module name="com.qulice.checkstyle.DiamondOperatorCheck"/>
     <module name="com.qulice.checkstyle.JavadocParameterOrderCheck"/>
     <module name="com.qulice.checkstyle.JavadocTagsCheck"/>
+    <module name="com.qulice.checkstyle.SingleLineCommentCheck">
+      <property name="format" value=" */\*[^*].*\*/ *"/>
+      <property name="message" value="This kind of comment is not allowed."/>
+      <property name="id" value="CStyleCommentNotAllowed"/>
+    </module>
+    <module name="com.qulice.checkstyle.SingleLineCommentCheck">
+      <property name="format" value=" */\*\* +[^A-Z\{ ][\W\s\S]*"/>
+      <property name="message" value="First sentence in a comment should start with a capital letter."/>
+      <property name="id" value="FirstLetterInCommentShouldBeCapital"/>
+    </module>
+    <module name="com.qulice.checkstyle.MultiLineCommentCheck">
+      <property name="format" value=" */\*\*\W+\* +[^A-Z\{ ][\W\s\S]*"/>
+      <property name="message" value="First sentence in a comment should start with a capital letter."/>
+      <property name="id" value="FirstLetterInCommentShouldBeCapital2"/>
+    </module>
+    <module name="com.qulice.checkstyle.MultiLineCommentCheck">
+      <property name="format" value=" */\*\* +[^A-Z\{ ][\W\s\S]*"/>
+      <property name="message" value="First sentence in a comment should start with a capital letter."/>
+      <property name="id" value="FirstLetterInCommentShouldBeCapital3"/>
```

Comment:
@PeJetuz I have some doubts about `id` properties here. Can we just omit them? Or at least to give them more meaningful names. `FirstLetterInCommentShouldBeCapital2` -> `FirstLetterInMultiLineCommenWithAsterisk`. Or something like this. `FirstLetterInCommentShouldBeCapital2` and `FirstLetterInCommentShouldBeCapital3` look strange to me.

Reply:
@volodya-lombrozo Thanks for your help with ArrayTrailingCommaCheck!
I renamed `id` and changed the message for this check. Thanks for pointing this out, I think the message should be more friendly now.